htdocs/mri/jiv/get_file.php

```diff
@@ -60,13 +60,6 @@
 // Now get the file and do file validation
 $File = $_GET['file'];
 
-// File validation
-if (strpos($File, ".") === false) {
-    error_log("ERROR: Could not determine file type.");
-    header("HTTP/1.1 400 Bad Request");
-    exit(3);
-}
-
```

Comment:
While this clearly isn't effective given that you were able to bypass it, is there any harm in keeping it in as a sanity check?